Automattic/jetpack
# Clear cache after a scheduler update is queued

Fixes: https://github.com/Automattic/wp-calypso/issues/90627
Context: p1715613932876249-slack-C01A60HCGUA

## Proposed changes:

* Force a cache clear the first ten seconds after a schedule creation, so that scheduled events at the same second will obtain a fresh cache and not previous value
* `jetpack_scheduled_update_updated` is run only on successful scheduled event creation

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add this branch to Jetpack Beta
* Quickly add/create/modify a scheduled update. Better when the clock is near second zero (08:00:57 -> 08:01:03, etc.)
* Ensure everything is ok
